kedro/framework/project/__init__.py

```diff
@@ -200,10 +200,10 @@ def __init__(self):
         # We suppress click here to hide tracebacks related to it conversely,
         # kedro is not suppressed to show its tracebacks for easier debugging.
         # sys.executable is used to get the kedro executable path to hide the top level traceback.
-        rich_traceback_install(
+        rich.traceback.install(
             show_locals=True, suppress=[click, str(Path(sys.executable).parent)]
         )
-        rich_pretty_install()
+        rich.pretty.install()
```

Comment:
@AntonyMilneQB I piggy-backed a trivial change that I wanted to suggest upon reviewing some of the new rich code. I think it's better to just use the original module path, rather than `from ... import ... as ...` and basically end up with the same thing, with underscores replacing dots.

Reply:
Very good suggestion, thank you!